TaxFoundation/spain-interactive
# Updated map with 2022 data

# Details
Data Updates to the Spain Interactive map for 2022.

#Steps to test
1. checkout branch
2. run 'npm install --legacy-peer-deps'
3. run 'npm start'
4. If map does not open automatically, go to address specified by npm start
5. ensure data on map is up to date
